### PR TITLE
adding link for module's example to documetation for the optuna.terminator module

### PR DESCRIPTION
**Pull request by HarshitNagpal29**
## Description of the changes
I added link for module's example to [documentation](https://optuna.readthedocs.io/en/stable/reference/terminator.html) for the optuna.terminator module as mentioned in the issue (https://github.com/optuna/optuna/issues/5204)

### CODE REVIEW

**Review by nzw0301 — CHANGES_REQUESTED** [review]
Thank you for sending the PR. 

As written in https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md#documentation, otpuna documentation uses reStructuredText, not Markdown, which I think you use in this PR. Could you use reStructuredText?  I suppose it will resolve the CI error.

In addition, tutorial links use a `seealso` block; I think it would be nice to use a similar syntax. Examples can be found in [this](https://github.com/optuna/optuna/blob/master/docs/source/reference/pruners.rst) or [this](https://github.com/optuna/optuna/blob/master/docs/source/reference/visualization/index.rst).

**Comment by HarshitNagpal29** [comment]
@nzw0301 Sir you can check this PR, I have made changes as suggested by, you can review it once, and tell me if anything else is required.

File: docs/source/reference/terminator.rst
Comment (HarshitNagpal29) [reply docs/source/reference/terminator.rst]: > Thank you for updating the PR. I'm sorry that my comment was not right because the linked page is not tutorial. So the following sentence would be enough.

NO problem, I have again made changes as suggested by you, please review it once.

File: docs/source/reference/terminator.rst
Comment (nzw0301) [inline docs/source/reference/terminator.rst:23]: My bad, sorry.
```suggestion
For an example of using this module, please refer to `this example <https://github.com/optuna/optuna-examples/tree/main/terminator>`_.
```

Comment (HarshitNagpal29) [reply docs/source/reference/terminator.rst]: > My bad, sorry.

No sir , it was completely my fault, I didn't notice that properly, sorry for inconvenience caused.
Sir, I can I contact you via e-mail? i want to discuss something.

Comment (nzw0301) [reply docs/source/reference/terminator.rst]: Thank you for updating the PR! 

> E-mail

I'm glad to hear that but I'm afraid I cannot. For optuna-related things, I'm not the right person because I'm not a member of the core developer team. But of course, I'm happy to support new contributors on Github like this PR's review :)

**Review by nzw0301 — APPROVED** [review]
Thank you for your quick reaction to my comments. I suppose the added sentence looks enough to resolve the original issue. 

I have two concerns. 

1. The place of the sentence. The sentence might be added to the end of L6 rather than the current position. I'd like to hear opinions from another reviewer of this PR (because I don't have permission to merge this PR). 
2. Give more concrete PR title.

File: docs/source/reference/terminator.rst
Comment (HarshitNagpal29) [reply docs/source/reference/terminator.rst]: > I'm glad to hear that but I'm afraid I cannot. For optuna-related things, I'm not the right person because I'm not a member of the core developer team. But of course, I'm happy to support new contributors on Github like this PR's review :)

No problem Sir, I will try discuss with someone, from core team member. Thanks for your acknowledgement.

**Comment by HarshitNagpal29** [comment]
> Thank you for your quick reaction to my comments. I suppose the added sentence looks enough to resolve the original issue. One concern is the location of the sentence.
> 
> The sentence might be added to the end of L6 rather than the current position. I'd like to hear opinions from another reviewer of this PR.

As you say, We can wait for their review and can proceed with their suggestions.

**Comment by HarshitNagpal29** [comment]
> 2\. Give more concrete PR title.

Sure.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Pull request merged**
